testsuite/domain/src/test/resources/host-configs/host-slave-elytron-no-legacy-realms.xml

```diff
@@ -0,0 +1,175 @@
+            </handlers>
+            <logger log-boot="true" log-read-only="false" enabled="false">
+                <handlers>
+                    <handler name="host-file"/>
+                </handlers>
+            </logger>
+            <server-logger log-boot="true" log-read-only="false" enabled="false">
+                <handlers>
+                    <handler name="server-file"/>
+                </handlers>
+            </server-logger>
+        </audit-log>
+        <management-interfaces>
+	  <native-interface sasl-authentication-factory="management-sasl-authentication">
+            <socket interface="management" port="${jboss.management.native.port:19999}"/>
+	  </native-interface>
+	  <http-interface http-authentication-factory="management-http-authentication">
+            <http-upgrade enabled="true" sasl-authentication-factory="management-sasl-authentication"/>
+            <socket interface="management" port="${jboss.management.http.port:19990}"/>
+	  </http-interface>
+        </management-interfaces>
+    </management>
+     <domain-controller>
+        <!-- Remote domain controller configuration with a host and port -->
+        <remote host="${jboss.test.host.master.address}" port="9999" authentication-context="slaveHostAContext"/>
+     </domain-controller>
+    <interfaces>
+      <interface name="management">
+	<inet-address value="${jboss.test.host.slave.address}"/>
+      </interface>
+        <interface name="public">
+            <inet-address value="${jboss.test.host.slave.address}"/>
+        </interface>
+        <interface name="unsecure">
+            <inet-address value="${jboss.test.host.slave.address}"/>
+         </interface>
+    </interfaces>
+    <jvms>
+        <jvm name="default">
+            <heap size="64m" max-size="128m"/>
+            <jvm-options>
+                <option value="-server"/>
+                <option value="-XX:MetaspaceSize=96m"/>
+                <option value="-XX:MetaspaceSize=96m"/>
+                <option value="-XX:MaxMetaspaceSize=256m"/>
+                <option value="--add-exports=java.base/sun.nio.ch=ALL-UNNAMED"/>
+            </jvm-options>
+        </jvm>
+    </jvms>
+
+    <servers>
+      <server name="server-one" group="main-server-group"/>
+      <server name="server-two" group="main-server-group" auto-start="true">
+        <socket-bindings port-offset="150"/>
+      </server>
+      <server name="server-three" group="other-server-group" auto-start="false">
+        <socket-bindings port-offset="250"/>
+      </server>
+     </servers>
+ 
+      <profile>
+        <subsystem xmlns="urn:jboss:domain:core-management:1.0"/>
+        <subsystem xmlns="urn:jboss:domain:jmx:1.3">
+            <expose-resolved-model/>
+            <expose-expression-model/>
+            <remoting-connector/>
+        </subsystem>
+	<subsystem xmlns="urn:wildfly:elytron:1.0" final-providers="combined-providers" disallowed-providers="OracleUcrypto">
+            <authentication-client>
```

Comment:
@darranl Is there an elytron only analog to using secret in security-identity in legacy auth-realms?